pyproject.toml

```diff
@@ -1,10 +1,155 @@
 [build-system]
 requires = [
-    # setuptools 42+ required for metadata.license_files support in setup.cfg
-    "setuptools>=42,<69",
+    # setuptools 61+ required for pyproject.toml support
+    "setuptools>=61,<69",
 ]
 build-backend = "setuptools.build_meta"
 
+# https://packaging.python.org/en/latest/specifications/declaring-project-metadata/
+[project]
+name = "torchgeo"
+description = "TorchGeo: datasets, samplers, transforms, and pre-trained models for geospatial data"
+readme = "README.md"
+requires-python = ">=3.9,<4"
+license = {file = "LICENSE"}
+authors = [
+    {name = "Adam J. Stewart", email = "ajstewart426@gmail.com"},
+]
+maintainers = [
+    {name = "Adam J. Stewart", email = "ajstewart426@gmail.com"},
+    {name = "Caleb Robinson", email = "Caleb.Robinson@microsoft.com"},
+    {name = "Isaac Corley"},
+    {name = "Nils Lehmann"},
+    {name = "Ashwin Nair"},
```

Comment:
@isaaccorley @nilsleh @ashnair1 I didn't add your emails because they aren't public on GitHub but let me know if you want me to add them. These names/emails are listed on the [PyPI project page](https://pypi.org/project/torchgeo/).

Reply:
n.lehmann@tum.de

Reply:
ashnair0007@gmail.com

Reply:
isaac.corley@my.utsa.edu